### ISSUE TITLE
Error: We couldn't find a Function or Program...

### ISSUE THREAD

**Issue by jhen0409** [codemix/babel-plugin-closure-elimination#5]
```
Error: test.js: We couldn't find a Function or Program...
    at Scope.getProgramParent (<path>/node_modules/babel-traverse/lib/scope/index.js:1058:11)
    at Scope.<anonymous> (<path>/node_modules/babel-traverse/lib/scope/index.js:651:25)
    at Scope.registerBinding (<path>/node_modules/babel-traverse/lib/scope/index.js:689:8)
    at Scope.crawl (<path>/node_modules/babel-traverse/lib/scope/index.js:905:14)
    at Scope.init (<path>/node_modules/babel-traverse/lib/scope/index.js:839:32)
    at NodePath.setScope (<path>/node_modules/babel-traverse/lib/path/context.js:150:30)
    at NodePath._containerInsert (<path>/node_modules/babel-traverse/lib/path/modification.js:112:12)
    at NodePath._containerInsertBefore (<path>/node_modules/babel-traverse/lib/path/modification.js:139:15)
    at NodePath.insertBefore (<path>/node_modules/babel-traverse/lib/path/modification.js:55:19)
    at PathHoister.run (<path>/node_modules/babel-plugin-closure-elimination/lib/index.js:137:16)
```

See the following code:
```js
import React, { Component } from 'react';

export class App extends Component {

  render() {
    return (
      <div>Hello</div>
    )
  }
};
```

My babelrc:
```json
{
  "presets": ["es2015", "stage-0", "react"],
  "plugins": ["closure-elimination"]
}
```

I used `babel-plugin-closure-elimination@1.0.0`.

**Comment by nicksrandall** [comment]
Can you send an example of what what your compiled code looks like for the above example?

**Comment by Gvozd** [comment]
I saw your example, and I think I have a solution
Look for solutions during the day

**Comment by Gvozd** [comment]
Havs some more troubles in your example
For example es2015 transform class to 
```javascript
var App = exports.App = function (_Component) {
  _inherits(App, _Component);

  function App() {
```
Has a three App(main variable, exports.App, App declaration)
Plugin must eliminate App declaration
But something wrong with renaming.
All three App renamed to _App, and exported value of your plugin becomes wrong

Perhaps it will need more time to solve

**Comment by Gvozd** [comment]
look at PR
https://github.com/codemix/babel-plugin-closure-elimination/pull/6

**Comment by jhen0409** [comment]
@Gvozd it works! Thanks! :)

**Comment by phpnode** [comment]
fixed in 1.0.1
